### PR TITLE
fmin numpy frontend

### LINKED ISSUE #13594: fmin
Add `fmin` to Numpy Frontend

### PR DESCRIPTION
**Pull request by AgnesLL**
Close #13594

### CODE REVIEW

**Comment by AgnesLL** [comment]
@karalleyna could you review this when you are free? Thanks.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

File: ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_extrema_finding.py [hunk 1]
```diff
@@ -344,3 +344,45 @@ def test_numpy_fmax(
         out=None,
         where=where,
     )
+
+
+# fmin
+@handle_frontend_test(
+    fn_tree="numpy.fmin",
+    dtype_and_inputs=helpers.dtype_and_values(
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_extrema_finding.py:352]: Hey here we will use `np_frontend_helpers.dtypes_values_casting_dtype` to generate the casting dtypes as well

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Would you solve the merge conflicts😊

**Comment by AgnesLL** [comment]
> Would you solve the merge conflicts😊

fixed

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @AgnesLL 
This function is a ufunc instance its testing should be handled the same way as other ufuncs, would you take a reference on `minimum` unit test.
Thanks

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**